topology/m4/mixercontrol.m4

```diff
@@ -30,7 +30,7 @@ define(`CONTROLMIXER_OPS',
 
 dnl C_CONTROLMIXER(name, index, ops, max, invert, tlv, KCONTROL_CHANNELS)
 define(`C_CONTROLMIXER',
-`SectionControlMixer."$1 PIPELINE_ID" {'
+`SectionControlMixer."$2 $1" {'
```

Comment:
This should be "PIPELINE_ID $1"?

Reply:
$2 is a variable which equals to PIPELINE_ID, the caller give the PIPELINE_ID to SectionControlMixer